src/openrct2/rct1/RCT1.h

```diff
@@ -437,174 +437,174 @@ namespace RCT1
 
     struct Peep : RCT12SpriteBase
     {
-        uint8_t pad_1F[3];
-        StringId name_string_idx;   // 0x22
-        uint16_t next_x;            // 0x24
-        uint16_t next_y;            // 0x26
-        uint8_t next_z;             // 0x28
-        uint8_t next_flags;         // 0x29
-        uint8_t outside_of_park;    // 0x2A
-        uint8_t state;              // 0x2B
-        uint8_t sub_state;          // 0x2C
-        PeepSpriteType sprite_type; // 0x2D
-        uint8_t type;               // 0x2E
+        uint8_t Pad1F[3];
+        StringId NameStringID;     // 0x22
+        uint16_t NextX;            // 0x24
+        uint16_t NextY;            // 0x26
+        uint8_t NextZ;             // 0x28
+        uint8_t NextFlags;         // 0x29
+        uint8_t OutsideOfPark;     // 0x2A
+        uint8_t State;             // 0x2B
+        uint8_t SubState;          // 0x2C
+        PeepSpriteType SpriteType; // 0x2D
+        uint8_t Type;              // 0x2E
```

Comment:
This is the source of your issues. Peep has two `type` variables. This PR changed it to one `Type` and one `type`.

Reply:
Thanks!